### PR TITLE
refactor: rename grid info to cell info

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- rename cell and grid
  - `cell[]` - > `row`
  - `GridInfo` -> `CellInfo`

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/types/time/datetime.d.ts [hunk 1]
```diff
@@ -8,8 +10,11 @@ type RawDate = {
   ms: number;
 };
 
-type GridInfo = {
-  day: number;
+interface CellStyleInfo {
```
Comment (adhrinae) [inline apps/calendar/types/time/datetime.d.ts:13]: 어차피 스타일 관련 속성만 남아있는데 `Info` 라는 단어까지 붙일 필요가 있을까요?

File: apps/calendar/types/time/datetime.d.ts [hunk 2]
```diff
@@ -8,8 +10,11 @@ type RawDate = {
...
   width: number;
   left: number;
-};
+}
+
+interface CellInfo extends CellStyleInfo {
```
Comment (adhrinae) [inline apps/calendar/types/time/datetime.d.ts:18]: 셀을 랜더링하는데 필요한 정보니까 Info 대신 다른 단어를 쓰고싶지만 얘는 일단 둬도 될거같네요.

**Review by adhrinae — COMMENTED** [review]
리뷰 완료합니다. 찾아서 고치는데 고생했어요.

File: apps/calendar/types/time/datetime.d.ts (hunk 1 above)
Comment (lja1018) [reply apps/calendar/types/time/datetime.d.ts:13]: 뭔가 관성적으로 `Info`를 붙이게 되는 것 같네요. 제거했습니다!

File: apps/calendar/types/time/datetime.d.ts (hunk 2 above)
Comment (lja1018) [reply apps/calendar/types/time/datetime.d.ts:18]: `Info` 대신 뭘 쓸지 논의해봐도 좋을 것 같습니다!

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**